readme.md

```diff
@@ -189,6 +189,7 @@ Other places you can look for MicroPython Libraries:
 * [ESP8266-Home-Assistant-Smart-Socket](https://github.com/AnthonyKNorman/ESP8266-Home-Assistant-Smart-Socket) - This MicroPython project is to hack a Hyleton313 cheap WiFi smart socket.
 * [ESP8266-Home-Assistant-RGB-Bulb](https://github.com/AnthonyKNorman/ESP8266-Home-Assistant-RGB-Bulb) - This MicroPython project is to hack a TYWE3S board in a cheap WiFi RGB Bulb.
 * [uPyIoT](https://github.com/lemariva/uPyIoT) - Connect an M5Stack ATOM running MicroPython to the Google Cloud Platform (GCP) to collect air-quality variables obtained from reading sensors.
+* [https://github.com/hilch/micropython-switchbot-thermometer-hygrometer] - read SwitchBot Thermometer/Hygrometer via Bluetooth
```

Comment:
The URL needs to be in round brackets () and there should be a title for the link in square brackets [] - please update 🙂

Reply:
argh, sorry :-)

```suggestion
* [micropython-switchbot-thermometer-hygrometer](https://github.com/hilch/micropython-switchbot-thermometer-hygrometer) - read SwitchBot Thermometer/Hygrometer via Bluetooth
```